UPGRADE-4.2.md

```diff
@@ -75,6 +75,11 @@ Process
    // alternatively, when a shell wrapper is required
    $process = Process::fromShellCommandline('ls -l');
    ```
+ * In `Bridge/Monolog` the methods `DebugProcessor::getLogs()`, `DebugProcessor::countErrors()`, `Logger::getLogs()` and `Logger::countErrors()` will have one `$request` argument in version 5.0, not defining it is deprecated since Symfony 4.2.
+ * In `Component/BrowserKit`, the `Client::submit()` method will have a new `$serverParameters` argument in version 5.0, not defining it is deprecated since version 4.2.
+ * In `Component/DomCrawler`, the `Crawler::children()` method will have one `$selector` argument in version 5.0, not defining it is deprecated since version 4.2.
+ * In `Component/Finder`, the `Finder::sortByName()` method will have one `$useNaturalSort` argument in version 5.0, not defining it is deprecated since version 4.2.
+ * In `Component/Serializer`, the `AbstractNormalizer::handleCircularReference()` method will have two more arguments : `$format` and `$context` in version 5.0, not defining it is deprecated since version 4.2.
```

Comment:
right now, all these lines are in the "Process" component section. They should be split each in their component's section instead. (same in UPGRADE-5.0.md)